src/main/proto/ga4gh/variants.proto

```diff
@@ -191,4 +191,29 @@ message Variant {
   // determination of genotype with respect to this variant. `Call`s in this
   // array are implicitly associated with this `Variant`.
   repeated Call calls = 12;
+
+  // The "variant_type" is used to denote e.g. structural variants.
+  // Examples:
+  //   DUP  : duplication of sequence following "start"; not necessarily in situ
+  //   DEL  : deletion of sequence following "start"
+  string variant_type = 13;
```

Comment:
+1 don't write an enum if someone else already has :).

Offering in the proto the variety of variants would be very helpful to other implementors.

```
duplication    (CURRENT_SVN)
SO Accession: 	SO:1000035 (SOWiki)

structural_variant    (CURRENT_SVN)
SO Accession: 	SO:0001537 (SOWiki)

insertion    (CURRENT_SVN)
SO Accession: 	SO:0000667 (SOWiki)
```